tests/snapshots/hacs-test-org/appdaemon-basic/test_update_entity_state.json

```diff
@@ -0,0 +1,42 @@
+{
+    "initial_state": {
+        "attributes": {
+            "auto_update": false,
+            "friendly_name": "Appdaemon Basic update",
+            "in_progress": false,
+            "installed_version": "1.0.0",
+            "latest_version": "1.0.0",
+            "release_summary": null,
+            "release_url": "https://github.com/hacs-test-org/appdaemon-basic",
+            "skipped_version": null,
+            "supported_features": 23,
+            "title": null
+        },
+        "context": {
+            "parent_id": null,
+            "user_id": null
+        },
+        "entity_id": "update.appdaemon_basic_update",
+        "state": "off"
+    },
+    "updated_state": {
+        "attributes": {
+            "auto_update": false,
+            "friendly_name": "Appdaemon Basic update",
+            "in_progress": false,
+            "installed_version": "1.0.0",
+            "latest_version": "1.0.0",
```

Comment:
The update entity is currently broken in experimental mode, this is fixed by https://github.com/hacs/integration/pull/3572